gematria/datasets/exegesis_benchmark.cc

```diff
@@ -291,10 +309,13 @@ int main(int Argc, char *Argv[]) {
           Benchmark::Latency, State, BenchmarkPhaseSelectorE::Measure,
           BenchmarkRunner::ExecutionModeE::SubProcess, 30, {}, Benchmark::Min));
 
-  if (pfm::pfmInitialize())
-    ExitOnErr(llvm::make_error<StringError>(inconvertibleErrorCode(),
-                                            "Failed to initialize libpfm"));
+  auto JsonMemoryBuffer = ExitOnErr(
+      errorOrToExpected(MemoryBuffer::getFile(AnnotatedBlocksJson, true)));
+
+  auto ParsedAnnotatedBlocks =
+      ExitOnErr(json::parse(JsonMemoryBuffer->getBuffer()));
```

Comment:
We should have a file error here too (IIUC, ``json::parse()` has no way of knowing the file name).